akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -504,7 +504,7 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
 
       case RecoveryTick(true) ⇒
         try onRecoveryFailure(
-          new RecoveryTimedOut(s"Recovery timed out, didn't get snapshot within $timeout s"),
+          new RecoveryTimedOut(s"Recovery timed out, didn't get snapshot within $timeout"),
```

Comment:
perhaps we should use `akka.util.PrettyDuration`? I think the FiniteDuration toString is not so readable, but I might be wrong (they might have improved that)?